src/main/kotlin/app/batch/DataKeyProcessor.kt

```diff
@@ -24,7 +25,7 @@ class DataKeyProcessor(val keyService: KeyService) : ItemProcessor<EncryptedStre
         }
         catch (e: DataKeyDecryptionException) {
             val message = "Failed to decrypt '${item.s3key}': '${e.message}'."
-            logger.error(message)
+            logError(logger, message)
```

Comment:
why the move from `logger.error` to `logError`?

Reply:
`logError` is the custom method that we are using to format messages as JSON I think